.stylelintrc.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Another file polluting the root of our file system. Can we move this to a folder? I can see us moving a bunch of files that aren't so important to everyone further down.

Reply:
Can we add `float`?

Reply:
<img width="1369" alt="Screen Shot 2020-10-27 at 12 25 12 PM" src="https://user-images.githubusercontent.com/10605724/97331269-8eab3500-184f-11eb-9bf2-855a63291f8e.png">

😢

Reply:
I can move the file but for some reason the vscode extension can't pick up files that aren't in the root. So if we move it we can't use it.

Reply:
I don't want us to use it actually. I see 2 notions of it in our css, we could drop those, or delay this request.